### DISCUSSION TITLE
Is the BigQuad deck compatible with Lighthouse positioning deck?

### DISCUSSION THREAD

**Discussion by GoLut** [bitcraze/discussions#355] [Q&A] [answered]
Hi all, 

**Question**: Is the BigQuad deck compatible with Lighthouse positioning deck?
I'm asking this question because the compatibility matrix given [here](https://www.bitcraze.io/documentation/system/platform/cf2-expansiondecks/#compatibility-matrixes), states:  2* "Will be supported in the future, SW update". 

Has this software update already occurred?

I'm doing research and would like to use the Crazyflie in both small and big configurations without having to buy 2 systems eg.  a Bolt 1.1 and a Crazieflie 2.1.

Thank you in advance!

**Comment by tobbeanton** [comment, marked as answer]
Yes they will work together but the UART1 used by the lighthouse deck is also wired to the BiqQuad deck "GPS" marked pins. So if the "GPS" marked pins are not used there should be no problem.

Later on the plan is to move the Lighthouse deck to I2C communication. This will also free up the "GPS" marked pins (UART1).

**Reply by GoLut** [reply]
Thank you for the fast reply!
 I do not intend to use a GPS for this project so all is good.
